bot/pkg/bsky/bsky.go

```diff
@@ -1 +1,104 @@
-package bsky
\ No newline at end of file
+package bsky
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/togdon/reply-bot/gsheets"
+)
+
+type Post struct {
+	URI         string                 `json:"uri"`
+	CID         string                 `json:"cid"`
+	Author      map[string]interface{} `json:"author"`
+	Record      map[string]interface{} `json:"record"`
+	ReplyCount  int                    `json:"replyCount"`
+	RepostCount int                    `json:"reposeCount"`
+	QuoteCount  int                    `json:"quoteCount"`
+}
+
+func FetchPosts(client *gsheets.GSheetsClient) {
+	BSKY_API_URL := "https://public.api.bsky.app/xrpc"
```

Comment:
not sure how you want to refactor this to use the feed instead. the final URL should be this:

```
https://public.api.bsky.app/xrpc/app.bsky.feed.getFeed?feed=at://did:plc:ltradugkwaw6yfotr7boceaj/app.bsky.feed.generator/aaapztniwbk46
```